components/managedidentity/04-managed-identities.tf

```diff
@@ -51,6 +52,60 @@ resource "azurerm_cosmosdb_sql_role_assignment" "identity_contributor" {
   scope              = data.azurerm_cosmosdb_account.version_reporter.id
 }
 
+data "azurerm_cosmosdb_account" "pipeline_metrics" {
+  count               = var.env == "prod" || var.env == "sandbox" ? 1 : 0
```

Comment:
We need this in every env, otherwise we won't be able to assign a managed identity access to it in aat for instance

Reply:
You'd need to a `cosmosdb_name` key:value for each sub then under `local. mi_cft` as thats whats being pulled in.